### PR TITLE
Add sensitivity analysis documentation

### PR DESCRIPTION
**Pull request by So-Fras**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

**What kind of change does this PR introduce?**
Documentation update.
This PR adds the sensitivity analysis documentation.

**Other information**:
Some links are still missing.

### CODE REVIEW

File: docs/simulation/sensitivity/configuration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+analysis results. The default value is 0.0.
+
+**voltageVoltageSensitivityValueThreshold**  
+
+The `voltageVoltageSensitivityValueThreshold` is the threshold under which sensitivity values having variable type
+`BUS_TARGET_VOLTAGE` and function type `BUS_VOLTAGE` will be filtered from the analysis results. The default value is 0.0.
+
+**flowVoltageSensitivityValueThreshold**  
+
+The `flowVoltageSensitivityValueThreshold` is the threshold under which sensitivity values having variable type among
```
Comment (annetill) [inline docs/simulation/sensitivity/configuration.md:22]: @vidaldid-rte I imagine I have to change this part to handle new sensitivities.

File: docs/simulation/sensitivity/configuration.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Configuration
+
+## Implementation
+
```
Comment (flo-dup) [inline docs/simulation/sensitivity/configuration.md:4]:
```suggestion
```

Comment (annetill) [reply docs/simulation/sensitivity/configuration.md:4]: Why?

Comment (flo-dup) [reply docs/simulation/sensitivity/configuration.md:4]: Isn't this a typo - there's no text in this section, and I don't see what we could have here in fact

File: docs/simulation/sensitivity/index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+## Introduction
+
+The sensitivity analysis module is dedicated to computing the linearized impact of small network variations on the state variables of some equipments.
+
+A sensitivity value is the numerical estimation of the partial derivative of the observed function with respect to the variable of impact.
+The sensitivity analysis can also be seen as the computation of partial derivatives on the network model.
+For example, it may be used to know, among a group of selected lines, which are the most impacted by a change in a generator production or a change of tap on a phase tap changer. The user story about [RSC capacity calculation](https://www.powsybl.org/pages/documentation/user/user-stories/capacity_calculation_rsc.html) provides an example of application of the sensitivity analysis.
+
+## Sensitivity analysis inputs
```
Comment (flo-dup) [inline docs/simulation/sensitivity/index.md:19]: To be in line with other pages
```suggestion
## Inputs
```

File: docs/simulation/sensitivity/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+      "id" : "l34",
+      "type" : "BRANCH"
+    } ]
+  } ]
+}
+```
+
+At the moment the only available sensitivity simulator officially compatible with PowSyBl is the one available through OpenLoadFlow. In this case, the network is provided only once in state N, and then all the calculations are done successively by modifying the Jacobian matrix directly in the solver based on the contingencies input. The network is thus loaded only once, which improves performance.
+
+## Sensitivity analysis outputs
```
Comment (flo-dup) [inline docs/simulation/sensitivity/index.md:100]:
```suggestion
## Outputs
```

File: docs/simulation/sensitivity/configuration.md (hunk 2 above)
Comment (annetill) [reply docs/simulation/sensitivity/configuration.md:4]: It's to put something like:
To use blabla for all sensitivity analyses, you have to configure the `sensitivity-analysis` module in your configuration file:
```yaml
sensitivity-analysis:
  default-impl-name: OpenLoadFlow
```

**Review by flo-dup — APPROVED** [review]

**Pull request merged**